packages/site-server/package.json

```diff
@@ -9,13 +9,15 @@
   "scripts": {
     "build": "wireit",
     "check": "wireit",
-    "serve:dev": "wireit"
+    "serve:dev": "wireit",
```

Comment:
`start` here and below?

Reply:
Sorry, should have pointed out, these were stacked PRs and the rename one is on top of this one.